src/main/resources/liquibase.xml

```diff
@@ -33,4 +33,16 @@
             </column>
         </createTable>
     </changeSet>
+    <changeSet id="202009140505" author="Neeraj/Aditi">
+        <addColumn tableName="support_requests">
+            <column name="support_request_id" type="VARCHAR(50)">
+                <constraints unique="true" nullable="false"/>
+            </column>
+            <column name="date_created" type="DATETIME" defaultValueDate="CURRENT_TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+            <column name="date_modified" type="DATETIME">
```

Comment:
Shouldn't we specify `defaultValueDate ` here as well?

Reply:
Done